<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="robot" default="/"/>
	<group ns="$(arg robot)">

		<!-- Motivation Manager  -->
		<node name="simulation_manager" pkg="motivational_dms" type="simulation.py" output="screen">
		</node>  

	</group>

</launch>